extensions/io/org.eclipse.smarthome.io.javasound/src/main/java/org/eclipse/smarthome/io/javasound/internal/JavaSoundAudioSink.java

```diff
@@ -118,4 +136,7 @@ private void runVolumeCommand(Closure closure) {
         }
     }
 
+    private boolean isMacOS() {
+        return System.getProperty("osgi.os").equals("macosx");
```

Comment:
Valid point. Again, this is the old code that was in place for long in openHAB 1/2.
I am not sure how to best identify a macos; in my tests, the non-mac implementation actually also worked on a mac, so I am not completely sure whether we need to do this separation at all. @kgoderis do you have any insights here?